pkg/config/config.go

```diff
@@ -676,6 +676,7 @@ func InitConfig(config Config) {
 	config.SetKnown("system_probe_config.offset_guess_threshold")
 	config.SetKnown("system_probe_config.enable_tcp_queue_length")
 	config.SetKnown("system_probe_config.enable_oom_kill")
+	config.SetKnown("system_probe_config.enable_tracepoints")
```

Comment:
is there a performance gain to be gained from enabling tracepoints ?  We intercept so few syscalls, it doesn't feel like its worth supporting two code paths (for kprobes and tracepoints)

Reply:
There is a theoretical performance gain. They also have a stable interface, whereas kprobes can technically change in any kernel version (example: the x64 indirection we have to handle). See https://lwn.net/Articles/748352/ for perf comparison (along with raw tracepoints).

They share the same actual probe code, just the entry function is different, which passes along the arguments.

If we ever change our kernel targets so that tracepoints are always available, IMHO it would be preferable to use them.

Reply:
I'm not opposed to adding select branches that offer increased performance for kernels after a particular version, if its available. I think this is a pattern we will (soon) be adopting widely as we support more use-cases via eBPF.

We'll definitely want to weigh the cost of supporting each additional path re: general testing and debugging of support cases. Since this is disabled by default, I think it's fine to keep it in while we further test before widely enabled based on detected kernel version.  

On the support side, we should probably add similar logging as bpftrace's `info` command to collect useful context:
- i.e https://github.com/iovisor/bpftrace/issues/1435#issue-664641877